esphome/components/sen5x/sen5x.h

```diff
@@ -18,19 +18,17 @@ enum ERRORCODE {
   UNKNOWN
 };
 
-// Shortest time interval of 3H for storing baseline values.
-// Prevents wear of the flash because of too many write operations
-const uint32_t SHORTEST_BASELINE_STORE_INTERVAL = 10800;
-// Store anyway if the baseline difference exceeds the max storage diff value
-const uint32_t MAXIMUM_STORAGE_DIFF = 50;
+enum RhtAccelerationMode : uint16_t {
```

Comment:
```suggestion
enum RhtAccelerationMode : uint8_t {
```

can this be uint8_t?

Reply:
Great catch! Here's how it's used:
```c
        uint16_t mode;
        if (this->read_data(mode)) {
          this->acceleration_mode_ = RhtAccelerationMode(mode);
        } else { // ...
```
I sort of assumed (but admittedly didn't test) that the compiler and/or clang-tidy will complain about narrowing the type, so I just left it alone. 🙈

Reply:
I guess they decided they might have 65535 modes 😆